tellor_all_languages/.env.example

```diff
@@ -0,0 +1,4 @@
+ETH_MNEMONIC="candy maple cake sugar pudding cream honey rich smooth crumble sweet treat"
```

Comment:
Would it not be better to place this information in an environment file?

Reply:
I guess you are correct. However, this is how Tellor provided the default template. Furthermore, I currently do not know if/how it is used. Hence, I would prefer not to modify it to keep it as close to their example as possible, such that it is easier for Tellor to follow up on this branch.